Cargo.toml

```
@@ -1,11 +1,11 @@
 [package]
 name = "taptap"
 description = "An implementation of the Tigo TAP protocol"
-authors = ["Will Glynn"]
-repository = "https://github.com/willglynn/taptap"
-keywords = ["solar"]
+authors = ["Will Glynn", "LiTinOveWeedle"]
+repository = "https://github.com/litinoveweedle/taptap"
```

With this PR targeted here, I'd rather:

```suggestion
repository = "https://github.com/willglynn/taptap"
```

---

First I made PR from main, which was dumb. So I can't easily update PR without breaking my min and I can't do that. If we close this PR here we will loose your comment, so my proposal is to solve those at least in discussion and then you can create your devel branch and I will open new PR from new branch as well. 

for the change:
OK, for sure, that was for my fork only.